test/parallel/test-buffer-write-noassert.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing buffer write functions
+
+function write(funx, args, result, res) {
+  {
+    const buf = Buffer.alloc(9);
+    assert.strictEqual(buf[funx](...args), result);
+    assert.deepStrictEqual(buf, res);
+  }
+
+  assert.throws(
+   () => Buffer.alloc(-1)[funx](0),
+   /^RangeError: "size" argument must not be negative$/
```

Comment:
Oh, yes, that's testing the wrong thing. It should test a legit alloc() call and an out-of-range call to the write function being tested. @larissayvette can you adjust it?